rslib/src/import_export/text/csv/metadata.rs

```diff
@@ -168,6 +161,16 @@ impl Collection {
                     metadata.guid_column = n;
                 }
             }
+            "limit dupe check to deck" => {
+                if let Ok(limit) = value.to_lowercase().parse() {
+                    metadata.limit_dupe_check_to_deck = limit;
+                }
+            }
+            "dupe resolution" => {
```

Comment:
Brainstorming - how about "if duplicate: update current/keep current/keep both"?

Reply:
Same here: Do we also change the ftl strings then?
By the way, the label says "Existing notes". An argument could be made to use that here as well.

Reply:
I don't mind 'existing notes' either, and it would save time. I am conscious that we're deciding on a file format here though and won't be able to change it later, so maybe it's worth the extra effort of bikeshedding the wording/updating the strings.

One other possible alternative:

`If matches:` instead of `If duplicate`. Thoughts? :-)

Reply:
Either is fine for, though we may want to align this with the naming above: If we go with `if matches`, I would also prefer `match scope`.

Reply:
Sounds good.